packages/templates/typescript/src/utils/get-result-type.ts

```diff
@@ -1,22 +1,16 @@
 import { pascalCase } from 'change-case';
 import { Field } from 'graphql-codegen-core';
 
-export function getResultType(type: Field, options: Handlebars.HelperOptions, skipPascalCase = false) {
-  const baseType = type.type;
-  const underscorePrefix = type.type.match(/^[\_]+/) || '';
+export function getFieldType(field: Field, realType: string, options: Handlebars.HelperOptions) {
```

Comment:
I extracted that because we need that logic in resolvers-template but without prefixes and stuff (things that convertedType has)